packages/tocco-ui/src/Icon/Icon.js

```diff
@@ -11,15 +11,19 @@ import {
 } from '../utilStyles'
 
 const getClassName = (icon, animation) => {
-  let cls = classNames({
-    'glyphicon': icon.startsWith('glyphicon-'),
-    'fa': icon.startsWith('fa-')
-  }, icon, 'icon')
+  let cls
```

Comment:
const getClassName = (icon, animation) => {
  if (icon) {
    return classNames({
      'glyphicon': icon.startsWith('glyphicon-'),
      'fa': icon.startsWith('fa-'),
      'fa-spin': animation === stylingAnimation.SPIN
  } else {
    return 'fa'
  }
}

or probably better with the ternary operator instead of the if